pyteal/ast/subroutine_test.py

```diff
@@ -128,20 +121,6 @@ def fnWithMixedAnns4AndIntReturn(a: Expr, b: ScratchVar) -> Bytes:
         ("mixed3 copacetic", fnWithMixedAnns3, [s, sv], None),
         ("mixed3 flipped", fnWithMixedAnns3, [sv, x], TealInputError),
         ("mixed3 missing the sv", fnWithMixedAnns3, [x, s], TealInputError),
-        (
-            "mixed3 missing the non-sv",
-            fnWithMixedAnns4AndIntReturn,
-            [sv, sv],
-            TealInputError,
-        ),
```

Comment:
Why get rid of this test?

Reply:
I moved thiis test case to `test_subroutine_definition_invalid()` because we are no longer allowing `Expr` subclasses for annotation return types.

Reply:
I also renamed the function `fnWithMixedAnns4AndBytesReturn` (it was wrongly named before)